src/cli_util.rs

```diff
@@ -1615,7 +1615,7 @@ fn expand_git_path(path_str: String) -> PathBuf {
     PathBuf::from(path_str)
 }
 
-fn resolve_op_for_load(
+pub fn resolve_op_for_load(
```

Comment:
Could be `pub(crate)` if we don't want that symbol in our public api?

Reply:
I don't mind having it in the public api for now. We should eventually go through the public api and think more carefully about what we expose, however.